web/packages/teleport/src/Audit/fixtures/index.ts

```diff
@@ -3062,6 +3062,7 @@ export const events = [
         updated_by: 'mike',
```

Comment:
Shouldn't we remove `updated_by` field from `members` in fixtures? `RawEventAccessList.members` doesn't contain it.